CSS/styles.css

```diff
@@ -322,48 +333,159 @@ ul {
   height: 20px;
 }
 
-.small-btn:hover {
-  background: #fff;
-  color: #1a1a1a;
-}
-
-.small-btn:active {
-  background: #98dede;
-  color: #1a2236;
-}
-
-.proj-btn {
-  margin: 1.25em 0 1.25em 1em;
-  padding: 9px 18px 9px 9px;
+.projbtn,
+.projbtn-mod,
+.projbtn-mod-src {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-image: url(../Images/arrow.png);
+  background-color: #1a2236;
+  background-repeat: no-repeat;
+  background-position: center right 12px;
+  background-size: 20px;
+  margin: 1em 0 1.25em 1.2em;
   box-sizing: border-box;
   cursor: pointer;
   height: 50px;
-  background: #1a2236;
+  width: fit-content;
+  padding-left: 2em;
+  padding-right: 5em;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
   text-align: center;
+  vertical-align: middle;
   color: #fff;
   border: 1px solid #fff;
 }
 
-.proj-btn img {
-  width: 20px;
-  height: auto;
-  margin-left: 10px;
+.projbtn-mod,
+.projbtn-mod-src {
+  background-image: url(../Images/ic_link.svg);
+  background-color: #fff;
+  color: #1a2236;
+  border: 1px solid #1a2236;
+  width: 15em;
+  margin: 0.75em 2em;
+}
+
+.projbtn-mod-src {
+  background-image: url(../Images/ic_github_white.svg);
 }
 
-.proj-btn:hover {
+.projbtn:hover {
   background: #fff;
   color: #1a2236;
 }
 
-.proj-btn:active {
+.projbtn-mod:hover,
+.projbtn-mod-src:hover {
+  background-image: url(../Images/ic_github.png);
+  background-color: #1a2236;
+  color: #fff;
+}
+
+.projbtn-mod:hover {
+  background-image: url(../Images/ic_download.png);
+}
+
+.projbtn:active {
   border: 2px solid #1a2236;
 }
 
+.modal-overlay {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  position: fixed;
+  top: 0;
+  left: -100%;
+  z-index: 99;
+  height: 100%;
+  width: 100%;
+  background-color: #fff;
+  opacity: 60%;
+  overflow-y: auto;
+  overflow-x: hidden;
+  transition: all 500ms ease-in-out;
+}
+
+.modal-overlay.active {
+  opacity: 100%;
+  left: 0;
+}
+
+.proj-wrp-mod {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  box-sizing: border-box;
+  background-color: #fff;
+  margin: 0;
+  max-height: 100%;
+  padding: 0;
+  width: 100%;
+}
+
+.projtxt-mod {
+  padding-top: 0.5em;
+  margin-bottom: 0.5em;
+  padding-left: 1.5em;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 24px;
+  color: #1a2236;
+}
+
+.projlang-mod {
+  margin-top: 12px;
+  margin-left: 45px;
+}
```

Comment:
- [ ] Kindly make sure the **Popup window** follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., spacing on text position, margins, and padding at the bottom,...). And also make sure you add the buttons at the bottoms ( Previous project && Next project) buttons to make your project stick to the Figma template.
#### Figma design
![pop_f](https://user-images.githubusercontent.com/83042742/174395739-b1873ebd-e142-4b2b-9ad9-c60591b4ffcd.png)
#### Your design
![f_s](https://user-images.githubusercontent.com/83042742/174396117-5db5db85-b4eb-472c-999e-bfcac027ec21.png)

Reply:
- [ ] Kindly as stated by previous reviewer please add appropriate padding and margin to match your design like Figma

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/174416510-2724c886-dc6d-420c-8f41-821b817c5645.png)  | ![image](https://user-images.githubusercontent.com/53402600/174416548-7e004c1e-0bfe-4aee-a893-7dcd423485a8.png) |